Resources/doc/format_listener.rst

```diff
@@ -36,7 +36,7 @@ Setting ``priorities`` to a non-empty array enables Accept header negotiations.
                 # setting fallback_format to null means that in case of a priority mismatch the next rule will be considered
                 - { path: '^/admin', methods: [ 'GET', 'POST'], priorities: [ 'xml', 'html'], fallback_format: ~, prefer_extension: false }
                 # you can specifically target the exception controller
-                - { path: '^/api', priorities: [ 'xml', 'json'], fallback_format: xml, attributes: [ _controller: FOS\RestBundle\Controller\ExceptionController ], prefer_extension: false }
+                - { path: '^/api', priorities: [ 'xml', 'json'], fallback_format: xml, attributes: { _controller: FOS\RestBundle\Controller\ExceptionController }, prefer_extension: false }
```

Comment:
By the way, was there a reason that we merged support for `attributes` in rules into the `master` branch and not in `1.8`? Looks to me like this would be backwards compatible.